pkg/kine/drivers/generic/generic.go

```diff
@@ -242,10 +241,6 @@ func Open(ctx context.Context, driverName, dataSourceName string, paramCharacter
 
 		FillSQL: q(`INSERT INTO kine(id, name, created, deleted, create_revision, prev_revision, lease, value, old_value)
 			VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)`, paramCharacter, numbered),
-
-		// Disable admission control for now to maintain backward-compatibility.
-		// TODO(MK-1397): Make policy and config configurable via CLI.
-		ac: &allowAllPolicy{},
```

Comment:
I'd leave this here by default, since the `generic` implementation will panic if this is nil

Reply:
Done